### ISSUE TITLE
Support OneGraph code exporter in Hasura console

### ISSUE THREAD

**Issue by coco98** [hasura/graphql-engine#2430]
What would it take for us to add support for the code exporter in the Hasura console? 

https://github.com/OneGraph/graphiql-code-exporter

https://www.onegraph.com/blog/2019/05/03_Filling_in_the_GraphQL_Pipeline_Ready_to_use_code_generation.html

@wawhal @rikinsk thoughts?

**Comment by rikinsk** [comment]
Should be pretty straight forward.

On a relate note. We also have to update to the newer explorer.

**Comment by coco98** [comment]
Yep! Could you create another issue for that as well? New explorer has some
nice conveniences :)

**Comment by rikinsk** [comment]
Explorer update tracked in #2313

**Comment by FluorescentHallucinogen** [comment]
PTAL at my build of GraphiQL + GraphiQL Explorer + GraphiQL Code Exporter (https://github.com/FluorescentHallucinogen/graphiql-online, https://graphiql-online.web.app). This can help.

**Comment by FluorescentHallucinogen** [comment]
Also it would be nice to integrate GraphQL Voyager into GraphiQL Online. Currently, it's already integrated into Console (see #2369).

Should I open another issue for that?

**Comment by FluorescentHallucinogen** [comment]
My issue #2128 was closed in favor of #314, but GraphiQL Online != Hasura Console.

The main idea of GraphiQL Online is that it's a standalone tool that useful for the whole GraphQL community, because it can be used with any GraphQL API (even by people who don't use Hasura at all).

**Comment by rikinsk** [comment]
@FluorescentHallucinogen reopened the issue. the idea was to wrap both under the same issue. seems like that didn't happen

**Comment by beerose** [comment]
Solved in https://github.com/hasura/graphql-engine/pull/4652
